modules/apps/commerce/commerce-product-asset-categories-web/src/main/resources/META-INF/resources/js/EditAssetCategoryCPDisplayLayout.js

```diff
@@ -0,0 +1,199 @@
+/**
+ * Copyright (c) 2000-present Liferay, Inc. All rights reserved.
+ *
+ * This library is free software; you can redistribute it and/or modify it under
+ * the terms of the GNU Lesser General Public License as published by the Free
+ * Software Foundation; either version 2.1 of the License, or (at your option)
+ * any later version.
+ *
+ * This library is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS
+ * FOR A PARTICULAR PURPOSE. See the GNU Lesser General Public License for more
+ * details.
+ */
+
+import {openSelectionModal} from 'frontend-js-web';
+
+export default function ({
+	assetCategory,
+	categorySelectorUrl,
+	itemSelectorUrl,
+	locale,
+	portletNamespace,
+}) {
+	const displayPageItemRemove = window.document.querySelector(
+		`#${portletNamespace}displayPageItemRemove`
+	);
```

Comment:
We should use `document.getElementById`. No need for `window`, `getElementById` is better than `querySelector` if we know we're querying by `id`. There is a lot of cases like this below.